README.md

```diff
@@ -1 +1,51 @@
-# spacetravelhub
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Space Travel Hub
+
+> A Web Appplication written with React and Redux which allows users to book rockets and join selected space missions.
+
+## Built With
+
+- React.js
+- Redux
+- JavaScript
+
+## Get Started
+To get a local copy of this calculator, Please follow these simple example steps.
+1. Open Git Bash.
+2. Change the current working directory to the location where you want the cloned directory. 
+ Ex: `C:User\your computer name\ cd <<your folder you want to clone to>>`
+4. Copy/Past this command :  `git clone https://github.com/alanluqman/spacetravelhub.git`  then press Enter
+5. Run this command `code .` to open this folder in VScode
+6. Open terminal and run this command `npm install`
+7. after finishing step 5, run this command `npm start` to run the project
+
```

Comment:
- [ ] Let's also add instructions to test project. It will make your project more easy to use.